concepticondata/conceptlists/Zgraggen-1980-380.tsv

```diff
@@ -308,40 +308,40 @@ Zgraggen-1980-380-374	2374	DREAM	374	dream
 Zgraggen-1980-380-375	3033	SCARED	375	be afraid
 Zgraggen-1980-380-380	1847	SICK	380	be sick
 Zgraggen-1980-380-381	1215	THOU	381	you
-Zgraggen-1980-380-382			382	our-dual
+Zgraggen-1980-380-382	3778	OUR (DUAL)	382	our-dual
 Zgraggen-1980-380-383	2310	US (OBLIQUE CASE OF WE)	383	us
-Zgraggen-1980-380-384	1213	YOU	384	you-pl-o2p
+Zgraggen-1980-380-384	3777	YOU (MORE THAN TWO OBLIQUE CASE)	384	you-pl-o2p
 Zgraggen-1980-380-385	2279	NO OR NOT	385	no-not
 Zgraggen-1980-380-387			387	negation-marker
 Zgraggen-1980-380-388	2680	MACHETE	388	kunai
-Zgraggen-1980-380-389	2274	YOUR (PLURAL)	389	your-pl
+Zgraggen-1980-380-389	3781	YOUR (MORE THAN TWO)	389	your-pl
 Zgraggen-1980-380-390	970	COCONUT	390	coconut
 Zgraggen-1980-380-391	2111	YOU TWO	391	you-dual
 Zgraggen-1980-380-392	2046	APRON	392	tanket
 Zgraggen-1980-380-393	732	YOUR (SINGULAR)	393	your-sg
 Zgraggen-1980-380-394	2642	HE OR SHE	394	he-she
-Zgraggen-1980-380-395	2400	THEIR	395	theirs
-Zgraggen-1980-380-396			396	your-pl-dual
+Zgraggen-1980-380-395			395	theirs
+Zgraggen-1980-380-396	2274	YOUR (PLURAL)	396	your-pl-dual
 Zgraggen-1980-380-397	2473	HIS (GENITIVE OF HE)	397	his
-Zgraggen-1980-380-398	2400	THEIR	398	their
-Zgraggen-1980-380-399			399	they-dual
+Zgraggen-1980-380-398	3780	THEIR (MORE THAN TWO)	398	their
+Zgraggen-1980-380-399	2635	THEY TWO	399	they-dual
 Zgraggen-1980-380-400	31	PAN	400	pan
-Zgraggen-1980-380-401			401	you-o2s
-Zgraggen-1980-380-402			402	we-dual
-Zgraggen-1980-380-403	780	OUR	403	our
+Zgraggen-1980-380-401	2304	THEE (OBLIQUE CASE OF YOU)	401	you-o2s
+Zgraggen-1980-380-402	2634	WE TWO	402	we-dual
+Zgraggen-1980-380-403	3779	OUR (MORE THAN TWO)	403	our
 Zgraggen-1980-380-404	984	WORK (LABOUR)	404	work-n
-Zgraggen-1980-380-405			405	you-pl
+Zgraggen-1980-380-405	3777	YOU (MORE THAN TWO OBLIQUE CASE)	405	you-pl
```

Comment:
YOU (MORE THAN TWO OBLIQUE CASE) is already mapped to you-pl-o2p in this list.

Reply:
Hm, and it cannot be subject case as there is already _ye_ in the list. I don't know what they mean here.

Reply:
One solution would be that _ye_ is YOU (dual or more than 2), you-pl is YOU (MORE THAN TWO) and you-pl-o2p is YOU (MORE THAN TWO OBLIQUE CASE). I changed it to that.

Reply:
That makes it inconsistent with their _we_ and _they_, though, which I mapped as more than 2 since they are only contrasted with duals.

Reply:
Hm, I looked into the actual book now, and it very much seems to me that the elicitation glosses we have here are not the same as in the book. I don't find most of the pronoun glosses there, only ye, we, thou, they, I, he. Instead it contains, _after the concept list_, paradigms distinguishing between free, possessive, genitive, verbal object, and either between singular, dual and plural (for all 3 persons) or between singular and plural (for all 3 persons) depending on the language.
So I have no idea where these glosses even come from.

Reply:
That was my impression too when I looked at the source. This might be the case that the list on concepticon was taken from elsewhere rather than being typed directly off the source